### PR TITLE
Cameras rosbridge

### PR DESCRIPTION
**Pull request by SijmenHuizenga**
* Split camera bridge things into it's own node
* Added a python script inbetween to launch a camera node and transform for each camera
* Changed some timeout behavior to fail faster when rpc connection cannot be made

### CODE REVIEW

File: ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp [hunk 1]
```diff
@@ -53,8 +53,18 @@ void AirsimROSWrapper::initialize_statistics()
...
+    try{
+        std::unique_lock<std::recursive_mutex> lck(car_control_mutex_);
+        state = airsim_client_.getRefereeState();
+        lck.unlock();
+    } catch (rpc::rpc_error& e)
+    {
+        std::string msg = e.get_error().as<std::string>();
+        std::cout << "Exception raised by the API, something went wrong while retreiving referee state for publishing track." << std::endl
+                  << msg << std::endl;
+    }
```
Comment (davidoort) [inline ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp:66]: thanks!!

File: ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp [hunk 2]
```diff
@@ -439,27 +449,37 @@ void AirsimROSWrapper::gps_timer_cb(const ros::TimerEvent& event)
  {
     std::cout << "error" << std::endl;
     std::string msg = e.get_error().as<std::string>();
-    std::cout << "Exception raised by the API:" << std::endl
+    std::cout << "Exception raised by the API while getting gps data:" << std::endl
```
Comment (davidoort) [inline ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp:452]: nice!

**Comment by davidoort** [comment]
Does this PR need https://github.com/FS-Online/Formula-Student-Driverless-Simulator/pull/103 to work?

File: ros/src/fsds_ros_bridge/include/statistics.h [hunk 3]
```diff
@@ -20,19 +20,16 @@ namespace ros_bridge
         void Print()
         {
             // print statistics summary to the console
-            std::cout << "-----------------------------------------------------\n";
```
Comment (davidoort) [inline ros/src/fsds_ros_bridge/include/statistics.h:23]: maybe the lines between messages are not required, but we should add lines between print outs for clarity. So at the end of the print function in the wrapper

**Comment by SijmenHuizenga** [comment]
> Does this PR need #103 to work?

no and yes. In this pr the new camera bridge is added. This camera bridge converts timestamps directly from unreal unix  to ros unix. The #103 pr updates the bridge to covert timestamps in the same way.

File: ros/src/fsds_ros_bridge/include/statistics.h (hunk 3 above)
Comment (SijmenHuizenga) [reply ros/src/fsds_ros_bridge/include/statistics.h:23]: added

**Comment by SijmenHuizenga** [comment]
@davidoort thanks for mentioning the color thing! It was indeed wrong. Updated color enconding and now it looks like this:
colors

**Pull request merged**